src/components/app/App.js

```diff
@@ -0,0 +1,30 @@
+// import ChirpsFeed from '../main/ChirpsFeed';
+import Footer from '../footer/Footer';
+import React from 'react';
+import {
+  BrowserRouter as Router,
+  Switch
+} from 'react-router-dom';
+import { getRoutes } from '../../routes';
+import Header from '../header/Header';
+import styles from 'styled-components';
+
+const AppStyle = styles.body`
```

Comment:
```suggestion
const AppStyle = styles.div`
```
This should not be 'body' because index.html already has a body tag and the entire app renders inside a div, inside that body tag. Browser warning: Body should not be nested in a <div>